test/mockserver/mockdatabaseadmin.ts

```diff
@@ -0,0 +1,205 @@
+/*!
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {google} from '../../protos/protos';
+import * as grpc from 'grpc';
+import * as protoLoader from '@grpc/proto-loader';
+import {createUnimplementedError, now} from './mockspanner';
+import v1 = google.spanner.admin.database.v1;
+import iam = google.iam.v1;
+import longrunning = google.longrunning;
+import Any = google.protobuf.Any;
+import Empty = google.protobuf.Empty;
+
+const PROTO_PATH = 'spanner_database_admin.proto';
+const IMPORT_PATH = __dirname + '/../../../protos';
+const PROTO_DIR =
+  __dirname + '/../../../protos/google/spanner/admin/database/v1';
+
+/**
+ * Load the Spanner Database Admin service proto.
+ */
+const packageDefinition = protoLoader.loadSync(PROTO_PATH, {
+  keepCase: false,
+  longs: String,
+  enums: String,
+  defaults: true,
+  oneofs: true,
+  includeDirs: [IMPORT_PATH, PROTO_DIR],
+});
+const protoDescriptor = grpc.loadPackageDefinition(packageDefinition);
+const databaseAdminProtoDescriptor =
+  protoDescriptor['google']['spanner']['admin']['database']['v1'];
+export const TEST_DATABASE_NAME =
```

Comment:
I think it is safe to remove `export` from this and next `const`. WDYT?

Reply:
Done